gtk-3.22/gtk-widgets.css

```diff
@@ -21,6 +21,7 @@
     -GtkTextView-error-underline-color: @error_color;
     -GtkIMHtml-hyperlink-color: @link_color;
     -GtkHTML-link-color: @link_color;
+    -GtkScrolledWindow-scrollbar-spacing: 0;
```

Comment:
It looks like instead of using the wildcard selector here that you can do

```
scrolledwindow {
    -GtkScrolledWindow-scrollbar-spacing: 0;
}
```

Reply:
yep... just wanted to keep less lines